packages/react-ds/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+
+import classNames from 'classnames/bind'
+
+import { IInput } from '@venice/core/models/input.model'
+import styles from '@venice/styles/components/Input.module.scss'
+
+const Input: FC<IInput> = ({
+  id,
+  label,
+  name,
+  style,
+  className,
+  error,
+  ...props
+}: IInput) => {
+  const selfId = id || `input__${name}`
```

Comment:
Using `input__${name}` we have a possibility to using multiples `id` with same name on HTML.

Just use `id` if using a prop for it or add a `uniqId` to generate it.

Reply:
An `id` should be unique, I think calling or adding `uniqId` is redundant. @felipefialho